website/docs/client/java/partials/java_transport.md

```diff
@@ -558,3 +558,145 @@ OpenLineageClient client = OpenLineageClient.builder()
 
 </TabItem>
 </Tabs>
+
+## [Composite](https://github.com/OpenLineage/OpenLineage/tree/main/client/java/src/main/java/io/openlineage/client/transports/CompositeTransport.java)
+
+The `CompositeTransport` is designed to aggregate multiple transports, enabling event emission to several destinations simultaneously. This is useful when events need to be sent to more than one target, such as a logging system and an API endpoint.
```

Comment:
Its current shape is not simultaneous. Isn't it?

Reply:
I may reword this - I meant you can send to same event to multiple locations at once.